### ISSUE TITLE
Add Duplicate Property validation Rule to a Menu Item on the ellipse of the root component

### ISSUE THREAD

**Issue by dubnemo** [OAGi/Score#1528]
For release 10.9.2, we ran into several duplicate property associations during the Release Validation batch process.   It would be very helpful to enable this specific validation earlier during the development process.

:  -> Check Duplicates on Where Used Components

@michaelfigura

**Comment by hakjuoh** [comment]
@kbserm During 10.9.2 release process, I found out that there were duplicate components as below.
<img width="512" alt="Screenshot 2023-07-28 at 2 06 57 PM" src="https://github.com/OAGi/Score/assets/16102116/4312ab44-d8a5-4359-ace2-6a6ec77ca0ed">

**Comment by dubnemo** [comment]
@hakjuoh correct.  But when Container Base was modified, it did not complain that a Where Used component was in conflict.  This check needs to be available in the UI at the time of re-basing, etc.
